app/assets/stylesheets/_star-radios.scss

```diff
@@ -23,14 +23,16 @@
     > label {
       color: #ddd;
       float: right;
+      margin: 0 5px;
+      font-size: 1.25em;
     }
 
     &.rating-options-group-enabled {
       /***** CSS Magic to Highlight Stars on Hover *****/
       > input:checked ~ label, /* show gold star when clicked */
       &:not(:checked) > label:hover, /* hover current star */
       &:not(:checked) > label:hover ~ label {
-        color: #FFD700;
+        color: $checked-star-color;
```

Comment:
I would leave it as it is. It is used in 2 places and if we decide to update start color to something new you only have to change single line of code ( it is router in mine "code-line-nazi" nature)